app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,98 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.github.dozermapper.core.Mapper;
+
+import java.util.List;
+
+/**
+ * 유저를 관리합니다.
+ */
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+    private final Mapper mapper;
+
+    @Autowired
+    public UserService(UserRepository userRepository, Mapper dozerMapper) {
+        this.userRepository = userRepository;
+        this.mapper = dozerMapper;
+    }
+
+    /**
+     * 유저 목록을 반환합니다.
+     *
+     * @return 유저 목록
+     */
+    public List<User> findAllUsers() {
+        return userRepository.findAll();
+    }
+
+    /**
+     * 주어진 id와 일치하는 유저를 반환합니다.
+     *
+     * @param id 유저 id
+     * @return 주어진 id와 일치하는 유저
+     * @throws UserNotFoundException '유저를 찾지 못했다'는 예외
```

Comment:
`@throws`에 대해서는 "~한 경우" 처럼 작성해 보세요.
```suggestion
     * @throws UserNotFoundException 유저를 찾지 못한 경우
```